fontir/src/variations.rs

```diff
@@ -1247,4 +1303,29 @@ mod tests {
             model.deltas(&point_seqs).unwrap()
         );
     }
+
+    #[test]
+    fn model_with_no_default_location() {
```

Comment:
The base master's location is at 0.0 (in normalized locations) by definition, otherwise it is simply not the base/default/neutral master. A VariationModel must have a base master and this has to be at 0.0, without one it's just invalid/incomplete. What's this and what does it have to do with variable kerning FEA?